wavenet/audio_reader.py

```diff
@@ -105,14 +169,22 @@ def thread_main(self, sess):
                 if self.sample_size:
                     # Cut samples into fixed size pieces
                     buffer_ = np.append(buffer_, audio)
-                    while len(buffer_) > self.sample_size:
+                    while len(buffer_) > 0:
```

Comment:
Hi @jyegerlehner , your argument is convincing! The remaining thing I concern is if we have `batch_size=2` for example, then different `sample_size`s will be a problem?